ui/admin/app/components/auth-methods/auth-method/actions/index.hbs

```diff
@@ -4,78 +4,64 @@
 }}
 
 {{#if (or @model.isOIDC @model.isLDAP)}}
-  <Rose::Dropdown
-    @text={{t (concat 'states.' @model.state)}}
-    @icon={{if
-      @model.isInactive
-      'flight-icons/svg/x-circle-16'
-      (if
-        @model.isPrivate
-        'flight-icons/svg/lock-16'
-        'flight-icons/svg/check-circle-16'
-      )
-    }}
-    @dropdownRight={{true}}
-    as |dropdown|
-  >
-    <dropdown.section @title={{t 'actions.change-state'}}>
-      <Rose::Form as |form|>
-        <form.radioGroup
-          @name='theme'
-          @selectedValue={{@model.state}}
-          @changed={{fn @changeState @model}}
-          as |radioGroup|
-        >
-          {{#each @model.options.state as |state|}}
-            <dropdown.item>
-              <radioGroup.radio
-                @label={{t (concat 'states.' state)}}
-                @value={{state}}
-              />
-            </dropdown.item>
-          {{/each}}
-        </form.radioGroup>
-      </Rose::Form>
-    </dropdown.section>
-  </Rose::Dropdown>
+  <Hds::Dropdown class='change-state' data-test-change-state as |dd|>
```

Comment:
the `<dd.Title @text={{t 'actions.change-state'}} />` seems to be missing

Reply:
oooh thanks for the catch, I will add that

Reply:
updated!